### PR TITLE
reorg 1.ngql-overview dir

### LINKED ISSUE #324: add pattern section
In case we forget.

### PR DESCRIPTION
**Pull request by whitewum**
to close #324.

### CODE REVIEW

File: docs-2.0/3.ngql-guide/1.nGQL-overview/3.graph-patterns.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+```ngql
+(a)-[*3..]->(b)
+(a)-[*]->(b)
+```
+
+## Assigning to path variables
+
+As described above, a series of connected vertices and edges is called a "path". nGQL allows
+paths to be named using an identifer, as exemplified by:
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/1.nGQL-overview/3.graph-patterns.md:164]: identifier

And it is more like a variable, right?

File: docs-2.0/3.ngql-guide/3.data-types/1.numeric.md [hunk 2]
```diff
@@ -10,4 +10,6 @@ An integer is declared with keyword `int` , which is 64-bit *signed*. The suppor
 
 ## Double floating point
 
-Double floating point data type is used for storing double precision floating point values. The keyword used for double floating point data type is `double`. There are no upper bounds and lower bounds.
+Double floating point data type is used for storing double precision floating point values. The keyword used for double floating point data type is `double`.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/3.data-types/1.numeric.md:13]: double floating-point data type
double-precision floating-point values

File: docs-2.0/3.ngql-guide/3.data-types/3.string.md [hunk 3]
```diff
@@ -12,3 +12,34 @@ nebula> CREATE TAG t1 (p1 FIXED_STRING(10)); -- Fixed length string type
 
 nebula> CREATE TAG t2 (p2 string); -- Variable length string type
 ```
+
+## OpenCypher Compatibility
+
+The openCypher requires that
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/3.data-types/3.string.md:18]: Finish the "that" clause?

**Review by randomJoe211 — COMMENTED** [review]
Great! 
Only some minor issues.

**Review by randomJoe211 — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**